### ISSUE TITLE
Consider improving your default messages

### ISSUE THREAD

**Issue by antonfirsov** [squad303/squad303app#16]
I believe this is a very important project, with a good potential, especially now that it has been advertised by Nexta.

However, I'm afraid with the current phrasing and tone of your default messages you will not be able to break through the mental walls of ordinary Russians being flooded by propaganda on TV.

I have some subjective ideas, but I'm not sure if they are good. You need a political communications expert on board who is familiar with the Russian atmosphere.

**Comment by fabriziopapa** [comment]
I have send this message:

Мне жаль, что мы не знаем друг друга, я итальянец, но я надеюсь, что эта война скоро закончится.  никто из нас не должен умирать и никто из нас не должен страдать.  те, кто страдает от войны, только мы.  простые люди, которые любят мир.

**Comment by antonfirsov** [comment]
Yeah maybe this can be turned into a feature idea, where users can type down their own ideas. The team can then cherry pick good ones.

My subjective recommendations:
- Aim for building sympathy. Without jumping over a sympathy threshold, the audience will perceive the message as spam.
- Don't demand people to overthrow the regime, it's completely unrealistic today. For now, the goal should be to give information.

Some specific points:

> Время свергнуть диктатора Путина!

See my comment above.

> Ищите правду о войне в Украине в TOR

99.9% of the audience has no idea what TOR is, and how does it work.

> топлива нет

I haven't asked people about this specific question, but I don't believe this is true, or at least the war didn't change anything here. Russia has big supplies of cheap fossil energy.

**Comment by 0xf00b** [comment]
IMHO this project should not be targeting smart urban youth, who probably already have access to media. 
instead, focus on older generation who are more fixed in their opinions. There maybe some material here:

https://papapover.com/
